### ISSUE TITLE
Create fake bounce / NDN

### ISSUE THREAD

**Issue by Fnordbert** [RealRaven2000/FiltaQuilla#268]
I have a 3letter-domain for which I have a "wildcard receiver account". Mail to all non-existent users at the domain end up here. This is very convenient to make up addresses for registrations at sites I don't trust etc. This account receives a lot of spam of course, and I'm well aware that it would be pointless to bounce these as the sender is usually fake.

From time to time however, there are some typos which are obviously intended for real humans, miss-typed CCs on long email chains etc.  I can't really be bothered to interact with these people and explain their mistakes, but it would be nice to be able to create a filter to respond with a fake Non-Delivery-Notice. (like they would get if I disabled the wildcard-account) It would not need to be a "real" NDN that would fool a Postmaster, it would only need to be good enough to make the senders get the point.

Would it be possible to do this with the current functionality, perhaps with SmartTemplate, or Javascript?
If not perhaps this could be added as a feature request?

**Comment by RealRaven2000** [comment]
Yes you can do an automatic reply (as long as Thunderbird is running to receive the emails) using the FiltaQuilla function "reply with SmartTemplate" - I highly recommend getting a SmartTemplates Pro license for this feature, see the feature comparison page of the "free as in beer" version vs Standard / Pro licenses:

https://smarttemplates.quickfolders.org/premium.html#filterForward

Note: I currently fund all development work in FQ through my commercial licenses for my other Add-ons, which is my full time job.

**Comment by Fnordbert** [comment]
Could you maybe give me some hints about how to best achieve the "NDN-look" with the mail as an attachment etc?
I had a suspicion it would be possible, but I must admit the learning curve and the cost is a little off-putting.
Total respect for the freemium model and all that, it's only that I'm looking for this solution just to "do the right thing" and be nice to some random strangers, so for this purpose it can't grow into too much of a project :-)

**Comment by RealRaven2000** [comment]
> Could you maybe give me some hints about how to best achieve the "NDN-look" with the mail as an attachment etc? I had a suspicion it would be possible, but I must admit the learning curve and the cost is a little off-putting. Total respect for the freemium model and all that, it's only that I'm looking for this solution just to "do the right thing" and be nice to some random strangers, so for this purpose it can't grow into too much of a project :-)

well, I am not an expert of what an NDN mail looks like, I think they usually are plain text emails, that return an error code and include some header detail from the original email. 

You can actually return almost all header data using SmartTemplates variables, such as `%to%` `%from%` `%subject%` etc. You would probably also not want to return the full body of the email so you could use a special command like `%quotePlaceholder(0)%` and suppress the quote header "on 12/09/2023 joe wrote:" using `%suppressQuoteHeaders%`. It might be an idea to open a separate issue on the [smartTemplates Github](https://github.com/RealRaven2000/SmartTemplates/issues) so we can spec it out there.

**Comment by Fnordbert** [comment]
Thanks! I will try it out, and open a ticket if I get stuck.
